doc/api.md

```diff
@@ -205,45 +205,45 @@ Note that there is a 1:1 correspondence between payload fields and DB fields (bu
+| `attributes`          | `active`             | List of active attributes of the device                                                                                                                                                                                                                                   | `[ { "name": "attr_name", "type": "Text" } ]` |
+| `lazy`                | `lazy`               | List of lazy attributes of the device                                                                                                                                                                                                                                     | `[ { "name": "attr_name", "type": "Text" } ]` |
+| `commands`            | `commands`           | List of commands of the device                                                                                                                                                                                                                                            | `[ { "name": "attr_name", "type": "Text" } ]` |
+| `internal_attributes` | `internalAttributes` | List of internal attributes with free format for specific IoT Agent configuration                                                                                                                                                                                         | LWM2M mappings from object URIs to attributes |
+| `static_attributes`   | `staticAttributes`   | List of static attributes to append to the entity. All the updateContext requests to the CB will have this set of attributes appended.                                                                                                                                    | `[ { "name": "attr_name", "type": "Text" } ]` |
+| `expressionLanguage`  | `expresionLanguage`  | optional boolean value, to set expression language used to compute expressions, possible values are: legacy or jexl. When not set or wrongly set, legacy is used as default value.                                                                                        |
+| `explicitAttrs`       | `explicitAttrs`      | Optional boolean value to support selective ignore of measures for device so that IOTA doesn’t progress. If not specified default is `false`. Additionally `explicitAttrs` can be used to define which meassures defined in JSON/JEXL array will be propagated to NGSI interface. [See all cases](advanced-topics.md#explicitly-defined-attributes-explicitattrs)                                                                            | `true/false`                                  |
```

Comment:
I think is better to describe it in a simplified way and provide the detailed information in the main section:

```suggestion
| `explicitAttrs`       | `explicitAttrs`      | optional field to support selective ignore of measures so that IOTA doesn’t progress. See details in [specific section](advanced-topics.md#explicitly-defined-attributes-explicitattrs)                                                                             | (see details in specific section)                                  |
```

Reply:
Fixed in 8a3b2ed949a7dac7be09931bcc000ac4f1bca90f